### PR TITLE
DEVELOPER-2708 - Changed the infographic PDF link and text as requested

### PR DESCRIPTION
**Pull request by Dantheman720**

### CODE REVIEW

File: solutions/enterprise-java/_common/solution.yml [hunk 1]
```diff
@@ -16,8 +16,8 @@ primary_promo:
   - title: Red Hat JBoss EAP
     description: "An application platform that provides an innovative modular, cloud-ready architecture; powerful management and automation; and world class developer productivity."
     links:
-      - name: "Infographic: Accelerate App development w/ EAP"
-        url: 'https://www.redhat.com/cms/public/Jaen%20Swart_Devops%20with%20EAP%20and%20Openshift.pdf'
+      - name: "Get started with EAP 7"
+        url: '/products/eap/get-started/'
```
Comment (paulrobinson) [inline solutions/enterprise-java/_common/solution.yml]: This won't work in staging-pr, it produces: http://developers-pr.stage.redhat.com/products/eap/get-started/

I think you need the 'local_url' key, not 'url'. See elsewhere in this file.

**Comment by Dantheman720** [comment]
Ah! I updated the yaml file.

**Comment by LightGuard** [comment]
retest this please

**Comment by LightGuard** [comment]
@Dantheman720 Who's the reviewer for this, is it @lincolnthree?

**Comment by Dantheman720** [comment]
@LightGuard - Yeah, I'm pretty sure it is.

**Comment by LightGuard** [comment]
@lincolnthree please verify

**Comment by Dantheman720** [comment]
https://issues.jboss.org/browse/DEVELOPER-2708

**Pull request merged**